ODCRequest/js/RequestForm.js

```diff
@@ -1,3 +1,5 @@
+var date_objects = []
```

Comment:
just out of curiosity, is there any special reason for this? rather than creating a class attribute that can be accessed with `this.date_objects` across all methods?

Reply:
You're right! Much better way of doing it